### ISSUE TITLE
error while trying to vendor kubevirt api

### ISSUE THREAD

**Issue by bharatnc** [kubevirt/kubevirt#2017]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug

**What happened**:

I was trying to leverage the kubevirt api in my gocode to create VMs on Kubernetes. I am making the import as ```	"github.com/kubevirt/kubevirt/pkg/api/v1"```. But when I run ``dep ensure`` (I use dep as my package management tool), I get the following error.

```            v0.3.0-alpha.2: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        v0.3.0-alpha.1: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        v0.3.0-alpha.0: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        v0.1.0-alpha: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.6: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.5: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.4: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.3: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.2: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.1: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        v0.0.1-alpha.0: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api": unable to read metadata: go-import metadata not found
        master: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1": unable to read metadata: go-import metadata not found
        cnv-1: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        release-0.10: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.11: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.12: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.13: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.14: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.4: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        release-0.6: unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/precond": unable to read metadata: go-import metadata not found
        release-0.8: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1": unable to read metadata: go-import metadata not found
        release-0.9: unable to deduce repository and source type for "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1": unable to read metadata: go-import metadata not found


```

It seems like that dep is unable to pull  "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1" and "kubevirt.io/kubevirt/pkg/precond"

**What you expected to happen**:

Not expect dep to fail.

**How to reproduce it (as minimally and precisely as possible)**:

Run ``dep init -v`` in a empty folder. Create a new go file and use the api to create a VMI

```
import (
	"github.com/kubevirt/kubevirt/pkg/api/v1"
)


func CreateNewVMIConfig() *v1.VirtualMachine {
	return &v1.VirtualMachine{
		Spec: v1.VirtualMachineSpec{
			Template: &v1.VirtualMachineInstanceTemplateSpec{
				Spec: v1.VirtualMachineInstanceSpec{
					Domain: v1.DomainSpec{
						Firmware: &v1.Firmware{},
						Features: &v1.Features{},
						Devices:  v1.Devices{},
					},
					Hostname: "",
					Networks: []v1.Network{},
					Volumes:  []v1.Volume{},
				},
			},
		},
	}
}
```
Try to run dep ensure -v to sync this new dependency that is imported.

**Anything else we need to know?**:

**Environment**: MacOS
- KubeVirt version (use `virtctl version`): NA
- Kubernetes version (use `kubectl version`): NA
- VM or VMI specifications: NA
- Cloud provider or hardware configuration: NA
- OS (e.g. from /etc/os-release): MacOS
- Kernel (e.g. `uname -a`): NA
- Install tools:NA
- Others: Golang version : go version go1.11.1 darwin/amd64

Thanks!

**Comment by petrkotas** [comment]
Hi @bharatnc KubeVirt uses glide as dependency manager for golang and currently we do not support dep.

We have tried dep in the past, but it was its early days and it did not worked properly. Therefore the project stuck with glide.

**Comment by cynepco3hahue** [comment]
@bharatnc please use import path `kubevirt.io/kubevirt/pkg/...` instead of `github.com/kubevirt/kubevirt/pkg/...`

**Comment by slintes** [comment]
I think this issue isn't related to dep vs glide, or using github.com in the import. It's a problem with the setup of the kubevirt.io website, and the meta data for go imports. There is some metadata on kubevirt.io:
```
$ curl -s kubevirt.io | grep go-import 
    <meta name="go-import" content="kubevirt.io/kubevirt git https://github.com/kubevirt/kubevirt">
    <meta name="go-import" content="kubevirt.io/containerized-data-importer git https://github.com/kubevirt/containerized-data-importer">
    <meta name="go-import" content="kubevirt.io/qe-tools git https://github.com/kubevirt/qe-tools">
    <meta name="go-import" content="kubevirt.io/node-recovery git https://github.com/kubevirt/node-recovery">
    <meta name="go-import" content="kubevirt.io/cluster-api-provider-external git https://github.com/kubevirt/cluster-api-provider-external">
````
But not on kubevirt.io/kubevirt (and other projects), as it should be (to my best knowledge).

Thx @bharatnc for reporting! Will try to figure out how to fix this, but it might need some time. Will update the status here asap.

**Comment by cynepco3hahue** [comment]
@slintes IIUC it cannot get `kubevirt.io/...` packages from the vendor, because it does not exist under the vendor, because the link wasn't correct (`github.com/kubevirt` instead of `kubevirt.io`), but let's wait for more information from @bharatnc :smile:

**Comment by bharatnc** [comment]
> @bharatnc please use import path `kubevirt.io/kubevirt/pkg/...` instead of `github.com/kubevirt/kubevirt/pkg/...`

@cynepco3hahue , that doesn't work:

```  etf-operator git:(bnc/etf-operator-tmpl) ✗ dep ensure -v
The following errors occurred while deducing packages:
  * "kubevirt.io/kubevirt/pkg/api/v1": unable to deduce repository and source type for "kubevirt.io/kubevirt/pkg/api/v1": unable to read metadata: go-import metadata not found

validateParams: could not deduce external imports' project roots
```

**Comment by bharatnc** [comment]
> I think this issue isn't related to dep vs glide, or using github.com in the import. It's a problem with the setup of the kubevirt.io website, and the meta data for go imports. There is some metadata on kubevirt.io:
> 
> ```
> $ curl -s kubevirt.io | grep go-import 
>     <meta name="go-import" content="kubevirt.io/kubevirt git https://github.com/kubevirt/kubevirt">
>     <meta name="go-import" content="kubevirt.io/containerized-data-importer git https://github.com/kubevirt/containerized-data-importer">
>     <meta name="go-import" content="kubevirt.io/qe-tools git https://github.com/kubevirt/qe-tools">
>     <meta name="go-import" content="kubevirt.io/node-recovery git https://github.com/kubevirt/node-recovery">
>     <meta name="go-import" content="kubevirt.io/cluster-api-provider-external git https://github.com/kubevirt/cluster-api-provider-external">
> ```
> But not on kubevirt.io/kubevirt (and other projects), as it should be (to my best knowledge).
> 
> Thx @bharatnc for reporting! Will try to figure out how to fix this, but it might need some time. Will update the status here asap.

@slintes, thank you!

**Comment by cynepco3hahue** [comment]
@bharatnc Can you please share your `Gopkg.toml` file?

**Comment by petrkotas** [comment]
@bharatnc sorry for not asking right away. How did you obtained the KubeVirt?
Where is your Kubevirt folder located on the disk, as well what is your $GOPATH ?

**Comment by bharatnc** [comment]
> @bharatnc sorry for not asking right away. How did you obtained the KubeVirt?
> Where is your Kubevirt folder located on the disk, as well what is your $GOPATH ?

Couldn't get ``github.com/kubevirt/kubevirt/pkg/api/v1`` using ``go get`` or ``dep`` so I just cloned the repo and put it here: ``~go/src/github.com/kubevirt``.

FYI:

Error while trying to fetch using``go get``

```
go get -v github.com/kubevirt/kubevirt/pkg/api/v1
github.com/kubevirt/kubevirt (download)
Fetching https://kubevirt.io/kubevirt/pkg/precond?go-get=1
Parsing meta tags from https://kubevirt.io/kubevirt/pkg/precond?go-get=1 (status code 404)
package kubevirt.io/kubevirt/pkg/precond: unrecognized import path "kubevirt.io/kubevirt/pkg/precond" (parse https://kubevirt.io/kubevirt/pkg/precond?go-get=1: no go-import meta tags ())
```

Also ``$GOPATH`` is : ``~/go``

**Comment by bharatnc** [comment]
> @bharatnc Can you please share your `Gopkg.toml` file?

@cynepco3hahue , nothing that much in the Gopkg.toml file.

I tried to do a fresh init using ``dep init -v``, added the ``main.go`` file and tried running the ``dep ensure -v`` command to sync the imports, ``Gopkg.toml`` , ``Gopkg.lock`` files and the vendor folder.

The ``main.go`` file:

```
package main 

import (
	"github.com/kubevirt/kubevirt/pkg/api/v1"
)

func main(){

}

func CreateNewVMIConfig() *v1.VirtualMachine {
	return &v1.VirtualMachine{
		Spec: v1.VirtualMachineSpec{
			Template: &v1.VirtualMachineInstanceTemplateSpec{
				Spec: v1.VirtualMachineInstanceSpec{
					Domain: v1.DomainSpec{
						Firmware: &v1.Firmware{},
						Features: &v1.Features{},
						Devices:  v1.Devices{},
					},
					Hostname: "",
					Networks: []v1.Network{},
					Volumes:  []v1.Volume{},
				},
			},
		},
	}
}

```

Also the Gopkg.toml file looks like this:

```
# Gopkg.toml example
#
# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
# for detailed Gopkg.toml documentation.
#
# required = ["github.com/user/thing/cmd/thing"]
# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
#
# [[constraint]]
#   name = "github.com/user/project"
#   version = "1.0.0"
#
# [[constraint]]
#   name = "github.com/user/project2"
#   branch = "dev"
#   source = "github.com/myfork/project2"
#
# [[override]]
#   name = "github.com/x/y"
#   version = "2.4.0"
#
# [prune]
#   non-go = false
#   go-tests = true
#   unused-packages = true


[prune]
  go-tests = true
  unused-packages = true

```

**Comment by slintes** [comment]
just did some tests with vanity urls, `go get` and go modules, where one imported package is configured on the root homepage only, and another one also on the package page.
Neither `go` get checks the root page, nor does `go build` on a module:

```
$ curl -s https://slintes.net/ | grep go-import
    <meta name="go-import" content="slintes.net/vanity1 git https://github.com/slintes/vanity1">
    <meta name="go-import" content="slintes.net/vanity2 git https://github.com/slintes/vanity2">

$ curl -s https://slintes.net/vanity1/ | grep go-import

$ curl -s https://slintes.net/vanity2/ | grep go-import
    <meta name="go-import" content="slintes.net/vanity2 git https://github.com/slintes/vanity2">

# this is on a go module, which imports both slintes.net/vanity1 and 2
$ go build -v
Fetching https://slintes.net/vanity2?go-get=1
Fetching https://slintes.net/vanity1?go-get=1
Parsing meta tags from https://slintes.net/vanity1?go-get=1 (status code 404)
go: slintes.net/vanity1@v0.0.0-20190212193600-a4a110cf7999: unrecognized import path "slintes.net/vanity1" (parse https://slintes.net/vanity1?go-get=1: no go-import meta tags ())
Parsing meta tags from https://slintes.net/vanity2?go-get=1 (status code 200)
get "slintes.net/vanity2": found meta tag get.metaImport{Prefix:"slintes.net/vanity2", VCS:"git", RepoRoot:"https://github.com/slintes/vanity2"} at https://slintes.net/vanity2?go-get=1
go: finding slintes.net/vanity2 v0.0.0-20190212193252-0bd7eacf9915
go: error loading module requirements

# this is a ordinary go get
$ go get -v slintes.net/vanity1
Fetching https://slintes.net/vanity1?go-get=1
Parsing meta tags from https://slintes.net/vanity1?go-get=1 (status code 404)
package slintes.net/vanity1: unrecognized import path "slintes.net/vanity1" (parse https://slintes.net/vanity1?go-get=1: no go-import meta tags ())

$ go get -v slintes.net/vanity2
Fetching https://slintes.net/vanity2?go-get=1
Parsing meta tags from https://slintes.net/vanity2?go-get=1 (status code 200)
get "slintes.net/vanity2": found meta tag get.metaImport{Prefix:"slintes.net/vanity2", VCS:"git", RepoRoot:"https://github.com/slintes/vanity2"} at https://slintes.net/vanity2?go-get=1
slintes.net/vanity2 (download)
```
I think we really need to configure https://kubevirt.io/<project> etc. to return a page with the needed go-import meta tag. Will check how to do that.

**Comment by slintes** [comment]
Argh.... forget all my comments. kubevirt.io is configured fine, I just missed a trailing slash with curl when testing... :/ Sorry.

**Comment by bharatnc** [comment]
> Argh.... forget all my comments. kubevirt.io is configured fine, I just missed a trailing slash with curl when testing... :/ Sorry.

@slintes , thanks, But still, when I do ``go-get`` on ``github.com/kubevirt/kubevirt/pkg/api/v1`` I get the following:

```
✗ go get github.com/kubevirt/kubevirt/pkg/api/v1
package kubevirt.io/kubevirt/pkg/precond: unrecognized import path "kubevirt.io/kubevirt/pkg/precond" (parse https://kubevirt.io/kubevirt/pkg/precond?go-get=1: no go-import meta tags ())

```
Is it possible that one / or ,ore packages inside of kubevirt isn't configured properly?

**Comment by bharatnc** [comment]
> > Argh.... forget all my comments. kubevirt.io is configured fine, I just missed a trailing slash with curl when testing... :/ Sorry.
> 
> @slintes , thanks, But still, when I do `go-get` on `github.com/kubevirt/kubevirt/pkg/api/v1` I get the following:
> 
> ```
> ✗ go get github.com/kubevirt/kubevirt/pkg/api/v1
> package kubevirt.io/kubevirt/pkg/precond: unrecognized import path "kubevirt.io/kubevirt/pkg/precond" (parse https://kubevirt.io/kubevirt/pkg/precond?go-get=1: no go-import meta tags ())
> ```
> Is it possible that one / or more packages inside of kubevirt isn't configured properly?

When I use glide to generate the vendor folder for the file that I'm using (incldued) above, seems to complain of the same issue: 

```
 test-kubvirt-import git:(master) ✗ glide up
[INFO]	Downloading dependencies. Please wait...
[INFO]	--> Fetching github.com/kubevirt/kubevirt
[INFO]	Resolving imports
[INFO]	--> Fetching updates for github.com/go-openapi/spec
[INFO]	--> Setting version for github.com/go-openapi/spec to v0.17.0.
[INFO]	--> Fetching updates for github.com/pborman/uuid
[INFO]	--> Fetching updates for k8s.io/api
[INFO]	--> Fetching updates for k8s.io/apimachinery
[INFO]	--> Setting version for k8s.io/apimachinery to kubernetes-1.12.4.
[INFO]	--> Fetching updates for k8s.io/client-go
[INFO]	--> Setting version for k8s.io/client-go to kubernetes-1.12.4.
[INFO]	--> Fetching updates for k8s.io/kube-openapi
[INFO]	--> Fetching updates for kubevirt.io/containerized-data-importer
[INFO]	--> Detected semantic version. Setting version for kubevirt.io/containerized-data-importer to v1.5.1
[INFO]	--> Fetching kubevirt.io/kubevirt/pkg/precond
[WARN]	Unable to checkout kubevirt.io/kubevirt/pkg/precond
[ERROR]	Error looking for kubevirt.io/kubevirt/pkg/precond: https://kubevirt.io/kubevirt/pkg/precond Not Found
```

**Comment by bharatnc** [comment]
These 2 are the problematic imports: https://github.com/kubevirt/kubevirt/blob/master/pkg/api/v1/types.go#L45-L46
```
       cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
	"kubevirt.io/kubevirt/pkg/precond"
```

**Comment by petrkotas** [comment]
@bharatnc @slintes I just did `go get -u -d kubevirt.io/kubevirt` and it works just fine. The site is configured properly.

The issue is, every bit of import in kubevirt library expects the path to be `$GOPATH/src/kubevirt.io/kubevirt` and not `$GOPATH/src/github.com/kubevirt`.
This is the reason why it does not work. 

@bharatnc please correct the paths and then try again.
Also please note, we really do not support dep as dependency manager at the moment.

**Comment by slintes** [comment]
> when I do `go-get` on `github.com/kubevirt/kubevirt/pkg/api/v1`

@bharatnc this is not intended to work, you do `go get` on the root of a project.

For dep that would mean (with the correct vanity url):
`dep ensure -add kubevirt.io/kubevirt`

But that also doesn't  work, because dep needs a Go file on the root of a project, see e.g. [1] (the workaround with `[[prune.project]]` and `non-go = false` doesn't work for me).

So we need to add a dummy Go file to KubeVirt. Will create a PR.

PS: I confirmed thar behaviour on a private project.

[1] https://github.com/golang/dep/issues/1306

**Comment by bharatnc** [comment]
> > when I do `go-get` on `github.com/kubevirt/kubevirt/pkg/api/v1`
> 
> @bharatnc this is not intended to work, you do `go get` on the root of a project.
> 
> For dep that would mean (with the correct vanity url):
> `dep ensure -add kubevirt.io/kubevirt`
> 
> But that also doesn't work, because dep needs a Go file on the root of a project, see e.g. [1] (the workaround with `[[prune.project]]` and `non-go = false` doesn't work for me).
> 
> So we need to add a dummy Go file to KubeVirt. Will create a PR.
> 
> PS: I confirmed that behaviour on a private project.
> 
> [1] [golang/dep#1306](https://github.com/golang/dep/issues/1306)

@slintes, yes that makes sense! Thanks a lot for the PR. If help is needed on #677, would be more than happy to contribute.

**Comment by slintes** [comment]
> If help is needed on #677, would be more than happy to contribute.

great :+1:  Someone on that issue volunteered to work on that, feel free to ask for the current status :slightly_smiling_face:

**Comment by bharatnc** [comment]
@slintes , just wanted to confirm - not sure if the following packages would need .go files to be present inside them:

```        "cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
	"kubevirt.io/kubevirt/pkg/precond"
```
For atleast one of the files, I get the following:

```bharatnc@thinkpad450s:~/go_workspace/src/kubevirt.io/kubevirt$ go get kubevirt.io/containerized-data-importer
package kubevirt.io/containerized-data-importer: no Go files in /home/bharatnc/go_workspace/src/kubevirt.io/containerized-data-importer
```
This we won't ned for ``"kubevirt.io/kubevirt/pkg/precond"`` as your PR is already adding .go file to the kubevirt.io/kubevirt prj directory.

**Comment by bharatnc** [comment]
> @slintes , just wanted to confirm - not sure if the following packages would need .go files to be present inside them:
> 
> ```
> 	"kubevirt.io/kubevirt/pkg/precond"
> ```
> For atleast one of the files, I get the following:
> 
> ```
> package kubevirt.io/containerized-data-importer: no Go files in /home/bharatnc/go_workspace/src/kubevirt.io/containerized-data-importer
> ```
> This we won't ned for `"kubevirt.io/kubevirt/pkg/precond"` as your PR is already adding .go file to the kubevirt.io/kubevirt prj directory.

@slintes , got an update:

When I try to vendor you branch using ``dep``, it almost succeeds, but gets stuck with this import:

```master: Could not introduce kubevirt.io/containerized-data-importer@master, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        debug-issue-335: Could not introduce kubevirt.io/containerized-data-importer@debug-issue-335, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        freeze-rh-summit: Could not introduce kubevirt.io/containerized-data-importer@freeze-rh-summit, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        release-v1.1: Could not introduce kubevirt.io/containerized-data-importer@release-v1.1, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        release-v1.2: Could not introduce kubevirt.io/containerized-data-importer@release-v1.2, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        release-v1.3: Could not introduce kubevirt.io/containerized-data-importer@release-v1.3, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        release-v1.4: Could not introduce kubevirt.io/containerized-data-importer@release-v1.4, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        release-v1.5: Could not introduce kubevirt.io/containerized-data-importer@release-v1.5, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
        untagged-f3a3ff3921b27eff32ca: Could not introduce kubevirt.io/containerized-data-importer@untagged-f3a3ff3921b27eff32ca, as its subpackage kubevirt.io/containerized-data-importer does not contain usable Go code (*build.NoGoError).. (Package is```

I think we should add a ``.go`` file in the root of that project as well. I think, then``dep`` would be happy to vendor ``kubevirt`` .

**Comment by bharatnc** [comment]
@slintes , verified that my intuition is correct. I have opened a PR in here: https://github.com/kubevirt/containerized-data-importer/pull/655, as I think the maintainers are different here.

TLDR;

To test, I added these additional stanzas into my ``Gopkg.toml`` file and then there was no problem vendoring :)

required = [
       "kubevirt.io/kubevirt/pkg/precond",
	"kubevirt.io/kubevirt/pkg/api/v1",
	"kubevirt.io/containerized-data-importer"
]

```
[[constraint]]
  name="kubevirt.io/kubevirt"
  branch="add-root-gofile-for-dep"

[[override]]
name = "kubevirt.io/containerized-data-importer"
source = "https://github.com/bharatnc/containerized-data-importer.git"
branch="bnc/add-go-file"
```

**Comment by slintes** [comment]
@bharatnc ack, thx for confirming. I created https://github.com/kubevirt/containerized-data-importer/pull/656

**Comment by slintes** [comment]
oh, bad timing, now we have 2 PRs. I will close mine, ok?

**Comment by slintes** [comment]
> oh, bad timing, now we have 2 PRs. I will close mine, ok?

done

**Comment by slintes** [comment]
`dep ensure -add kubevirt.io/kubevirt` works for me now
@bharatnc can you confirm?
